src/robusta/core/model/nodes.py

```diff
@@ -0,0 +1,94 @@
+from pydantic.main import BaseModel
+from typing import List, Dict
+
+
+class NodeInfo(BaseModel):
+    name: str
+    node_creation_time: str
+    internal_ip: str
+    external_ip: str
+    taints: str
+    conditions: str
+    memory_capacity_mb: int
+    allocatable_memory_mb: int
+    allocated_memory_mb: int
+    cpu_capacity: float
+    allocatable_cpu: float
+    allocated_cpu: float
```

Comment:
I would consider storing this in a more generic format, like:

```
resources["cpu"] = ResourceStats(allocated, allocatable, capacity)
```

It's not a must, but this is likely to expand in the future.

Reply:
Not sure why it matters, but ok

Reply:
I'm skipping this change, because cpu stats are float and mem is int